dapp/src/utility/sigmadao.utility.ts

```diff
@@ -37,3 +38,35 @@ export const validateFundAmount = (
 		return Promise.resolve();
 	}
 };
+
+/**
+ * Validate the Algorand Address
+ * @param _rule the antd rules
+ * @param addr Address
+ */
+export const validateAlgroandAddress = (
+	_rule: Rule,
+	addr: string,
+) => {
+	if (!addr) {
+		throw new Error(
+			`Please input the address.`
+		);
+	}
+	if (!isValidAlgroandAddress(addr)) {
+		throw new Error(
+			`Please provide valid address.`
+		);
+	}
+};
+
+/**
+ * Verify Algorand address
+ * @param addr Address to be verifited
+ */
+export const isValidAlgroandAddress = (addr: string): boolean => {
```

Comment:
I think we already have a function for it in algob, or we can just use one from algosdk

Reply:
Yes, decodeAddress method is from algosdk. See the body of method.

Reply:
Thx, I have used the method from algosdk.